testing/eigen/APKBUILD

```diff
@@ -0,0 +1,27 @@
+# Contributor: Bradley J Chambers <brad.chambers@gmail.com>
+# Maintainer: Bradley J Chambers <brad.chambers@gmail.com>
+pkgname=eigen
+pkgver=3.3.4
+pkgrel=0
+pkgdesc="Eigen is a C++ template library for linear algebra"
+url="http://eigen.tuxfamily.org/index.php?title=Main_Page"
+arch="all"
+license="MPL2"
+depends=""
+makedepends=""
+install=""
+subpackages="$pkgname-dev"
+source="$pkgname-$pkgver.tar.gz::http://bitbucket.org/eigen/$pkgname/get/$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-eigen-5a0156e40feb"
+
+build() {
+        return 0
+}
+
+package() {
+        mkdir -p "$pkgdir"/usr/include
+        cp -r "$builddir"/Eigen "$pkgdir"/usr/include
+        cp -r "$builddir"/unsupported "$pkgdir"/usr/include
```

Comment:
Nope, it doesn't belong inside `/usr/include`. That was my mistake. Now, both `Eigen` and `unsupported` are installed to `/usr/include/eigen3`. For more on `unsupported` see [here](https://eigen.tuxfamily.org/dox/unsupported/index.html).